buildchow/templates/defaults.yaml

```diff
@@ -4,35 +4,27 @@
     default-jdk: jdk1.8
     default-maven-version: maven-3.2.5
     default-remote: origin
-    jvm-memory: "-Xms512m -Xmx1024m -XX:PermSize=128m -XX:MaxPermSize=256m"
+    default-releng-publish-zip: "org.jboss.tools.releng:jbosstools-releng-publish:4.3.0.CR1-SNAPSHOT:zip"
+    default-daysToKeep: 10
+    default-numToKeep: 10
+    default-artifactDaysToKeep: 10
+    default-artifactNumToKeep: 10
+    default-nodes: "(RHEL6||RHEL7||beaker||jboss-prod)&&!ia64&&!ppc64&&!rhts"
+    # since we use JDK 8 now, no need for these:  -XX:PermSize=128m -XX:MaxPermSize=256m"
+    jvm-memory: "-Xms512m -Xmx1536m"
     tests-folder: tests # for some it is all-tests
     component-responsible: ""
     scm-schedule: H H/8 * * * # every 8 hours starting a random hour. Removes need for each job to configure it (http://htmlpreview.github.io/?https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html)
-    download-cache:  "$JENKINS_HOME/static_build_env/jbds/download-cache"
-    maven-settings: "$JENKINS_HOME/../.m2/local-settings.xml" # need more reliable location
+    # these paths only exist on NFS-mounted slaves in the JBoss Jenkins cluster in mw.lab.eng.bos.redhat.com, which have a hudson user on them
+    # some slaves are configured with JENKINS_HOME = /qa/hudson_master/hudson_home/hudson_workspace, which is NOT the path we want here 
+    HUDSON_USER_HOME: "/home/hudson"
+    download-cache:  "{HUDSON_USER_HOME}/static_build_env/jbds/download-cache"
```

Comment:
/static_build_env only exists for some Jenkins slaves. Unless you plan to recreate it on your local Jenkins, it won't be there and won't work in $JENKINS_HOME.

Note too that based on https://engineering.redhat.com/rt/Ticket/Display.html?id=356526 not all slaves use /home/hudson so we should be using the ${HOME} folder of the user running Jenkins, which is what I set above. 

I still think this value should be externalized to the server config somehow.

Also not sure what to do about the reference to "local-settings.xml", which I don't see in our current jobs.